### PR TITLE
Fix path for key.pem and cert.pem

### PR DESCRIPTION
**Pull request by greenbender**
Changes:
  * Corrected home dir for the news user
  * Cleaned up unnecessary RUN commands
  * Execute the entrypoint as root so that cert creation can write to /etc/news/ then drop privileges
  * Tweak the innd start process, more closely matching how launch would occur if innd was started using `service inn start`.
  * Added `rc.news.local` to launch the additional `nnrpd` process required to handle implicit TLS connections. Given the launch process now uses `rc.news` this was the best/easiest way to do this. NOTE: I used the same bash scripting style used in `rc.local` when creating this file.

### CODE REVIEW

**Comment by greenbender** [comment]
I see you spotted the sll path bug... I had that in this PR too, but you beat me to it.

File: bin/rc.news.local [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#! /bin/sh
+
+## Additional bootscript used add TLS support
+## This	script will	be automatically run by	rc.news	to perform start and stop
+## actions
```
Comment (cclauss) [inline bin/rc.news.local:5]:
```suggestion
## This script will be automatically run by rc.news to perform start and stop actions
```

File: Dockerfile [hunk 2]
```diff
@@ -2,28 +2,24 @@
...
-  sed -i 's/#\(tlscertfile:\)/\1/' etc/news/inn.conf && \
-  sed -i 's/#\(tlskeyfile:\)/\1/' etc/news/inn.conf && \
-  mkdir -p /var/run/news && \
-  touch /var/run/news/nnrpd-563.pid && \
-  chown news:news /var/run/news/nnrpd-563.pid && \
+  usermod -d /usr/lib/news news && \
   mkdir -p /run/news && \
   chown news:news /run/news
 
+ENV PATH=/usr/lib/news/bin:$PATH
```
Comment (greenbender) [inline Dockerfile:16]: Not really required, Just for convienience when if `exec`d into the container.

File: bin/rc.news.local [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+NNRPD=${NEWSBIN}/nnrpd
+NNRPD_TLS_PORT=563
+
+case x"$1" in
+Xstop)
+    # Stop TLS nnrpd if	its	running
+    if [ -f	${PATHRUN}/nnrpd-${NNRPD_TLS_PORT}.pid ]; then
```
Comment (cclauss) [inline bin/rc.news.local:13]:
```suggestion
    # Stop TLS nnrpd if it is running
    if [ -f ${PATHRUN}/nnrpd-${NNRPD_TLS_PORT}.pid ]; then
```

File: bin/rc.news.local [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        kill $(cat ${PATHRUN}/nnrpd-${NNRPD_TLS_PORT}.pid)
+        rm -f ${PATHRUN}/nnrpd-${NNRPD_TLS_PORT}.pid
+    fi
+    exit 0
+    ;;
+esac
+
+# Start	TLS	nnrpd 
+${NNRPD} -D	-p ${NNRPD_TLS_PORT} -S
```
Comment (cclauss) [inline bin/rc.news.local:22]:
```suggestion
# Start TLS nnrpd 
${NNRPD} -D -p ${NNRPD_TLS_PORT} -S
```

File: Dockerfile [hunk 5]
```diff
@@ -2,28 +2,24 @@
 FROM debian:bookworm-slim
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN usermod -d /etc/news news
-COPY --chown=news:news etc/* etc/news/
-COPY --chown=news:news db/* db/
+COPY etc/* etc/news/
+COPY bin/* /usr/lib/news/bin/
```
Comment (greenbender) [inline Dockerfile:6]: No need to chown these. Keep `root` ownership for consistency with other files.

File: Dockerfile [hunk 6]
```diff
@@ -2,28 +2,24 @@
...
 
+
 COPY docker-entrypoint.sh /docker-entrypoint.sh
 ENTRYPOINT ["tini", "--", "/docker-entrypoint.sh"]
-CMD ["-f"]
 
 ENV PATH=/usr/lib/news/bin:$PATH
-VOLUME /var/lib/news/db
-USER news
+VOLUME /var/lib/news
```
Comment (greenbender) [inline Dockerfile:21]: There is no db dir the db lives directly in `/var/lib/news`

File: bin/rc.news.local (hunk 4 above)
Comment (greenbender) [reply bin/rc.news.local:21]: Yup my tab/spaces were misconfigured... whoops.

**Comment by greenbender** [comment]
There are some more stray tabs in docker-entrypoint.sh, but they are consistent with the existing tabs in the key generation block. So I'm just going to leave them for now.

**Comment by cclauss** [comment]
`/docker-entrypoint.sh: 3: .: cannot open lib/news/innshellvars: No such file`

**Comment by greenbender** [comment]
> % `docker build --no-cache --progress plain -t inn . && docker run --rm -it -p119:119 -p563:563 inn`
> 
> ```
> /docker-entrypoint.sh: 3: .: cannot open lib/news/innshellvars: No such file
> ```

Resolved... I tried bringing in some changes from your repo on merge conflict and things got a little wacky.

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
Works on my machine.  Please give me a thumbs up when you are ready.

**Pull request merged**